### PR TITLE
rn,tracks: fix loadEffects

### PR DESCRIPTION
**Pull request by saghul**
The resolved promise is expected to return an array.

### CODE REVIEW

File: react/features/base/tracks/loadEffects.native.js [hunk 1]
```diff
@@ -7,5 +7,5 @@
  * @returns {Promsie} - A Promise which resolves when all effects are created.
  */
 export default function loadEffects(store: Object): Promise<any> { // eslint-disable-line no-unused-vars
-    return Promise.resolve();
+    return Promise.resolve([]);
```
Comment (paweldomas) [inline react/features/base/tracks/loadEffects.native.js]: maybe worth updating JSDoc for the return type?

Comment (paweldomas) [reply react/features/base/tracks/loadEffects.native.js]: would flow help here?

Comment (saghul) [reply react/features/base/tracks/loadEffects.native.js]: good point, i'll fix it in a jiffy

**Comment by saghul** [comment]
@paweldomas updated PTAL.

**Pull request merged**